lib/ansible/modules/cloud/google/gcp_container_cluster.py

```diff
@@ -0,0 +1,1036 @@
+              "nodeConfig") will be used to create a "NodePool" object with an
+              auto-generated name. Do not use this and a nodePool at the same
+              time.
+        required: true
+    node_config:
+        description:
+            - Parameters used in creating the cluster's nodes.
+            - For requests, this field should only be used in lieu of a
+              "nodePool" object, since this configuration (along with the
+              "initialNodeCount") will be used to create a "NodePool" object
+              with an auto-generated name. Do not use this and a nodePool at the
+              same time. For responses, this field will be populated with the
+              node configuration of the first node pool. If unspecified, the
+              defaults are used.
+        required: false
+        suboptions:
+            machine_type:
+                description:
+                    - The name of a Google Compute Engine machine type (e.g.
+                    - n1-standard-1).  If unspecified, the default machine type is
+                      n1-standard-1.
+                required: false
+            disk_size_gb:
+                description:
+                    - Size of the disk attached to each node, specified in GB. The
+                      smallest allowed disk size is 10GB. If unspecified, the
+                      default disk size is 100GB.
+                required: false
+            oauth_scopes:
+                description:
+                    - The set of Google API scopes to be made available on all of
+                      the node VMs under the "default" service account.
+                    - |
+                      The following scopes are recommended, but not required, and by
+                      default are not included:
+                      https://www.googleapis.com/auth/compute is required for
+                      mounting persistent storage on your nodes.
+                    - |
+                      https://www.googleapis.com/auth/devstorage.read_only is
+                      required for communicating with gcr.io (the Google Container
+                      Registry).
+                    - If unspecified, no scopes are added, unless Cloud Logging or
+                      Cloud Monitoring are enabled, in which case their required
+                      scopes will be added.
+                required: false
+            service_account:
+                description:
+                    - The Google Cloud Platform Service Account to be used by the
+                      node VMs.  If no Service Account is specified, the "default"
+                      service account is used.
+                required: false
+            metadata:
+                description:
+                    - The metadata key/value pairs assigned to instances in the
+                      cluster.
+                    - |
```

Comment:
This seems to only happen on second-paragraph and beyond rendering, if that helps you track it down.